torch/_dynamo/output_graph.py

```diff
@@ -1936,6 +1936,9 @@ def __init__(self, output_graph, parent=None, is_export=False, source_target=Non
         # backward recomputation of the checkpoint region doesn't affect its correctness.
         self.allow_side_effects_under_checkpoint = False
 
+        # True if this tracer is currently tracing (reconstructing) into a Python generator
+        self.in_generator = False
```

Comment:
Is this just for reconstructing, or also for tracing?
If it's just for reconstructing then we should name it is_reconstructing_generator

Reply:
Looks like just reconstructing, so let's rename it.